lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2537,6 +2537,18 @@ definitions:
       ues_connected:
         type: integer
         format: int32
+      gps_altitude:
+        type: string
+      vendor:
+        type: string
+      model_name:
+        type: string
+      rf_state:
+        type: boolean
```

Comment:
An Enum can be used and adds some clarity here. What does `rf_state` = `true` mean?